modules/cnv-updating-access-mode-to-enable-live-migration.adoc

```diff
@@ -0,0 +1,30 @@
+// Module included in the following assemblies:
+//
+// * cnv/live_migration/cnv-live-migration.adoc
+
+[id="cnv-updating-access-mode-to-enable-live-migration_{context}"]
+= Updating access mode to enable LiveMigration
+
+Storage class defaults in {VirtProductName} are configured to provide ReadWriteOnce (RWO) access mode
+by default. Therefore, LiveMigration is not enabled by default unless you
+alter the access mode of your virtual machine.
+
+.Prerequisites
+* In order to use LiveMigration, virtual machines must have a persistent volume claim (PVC) with a shared
```

Comment:
While this is required to live migrate a VM, I don't think it's a prerequisite for updating a ConfigMap

Reply:
I removed this prereq in favor of the prereq in the assembly with the cross-ref described above.